### PR TITLE
Migration to null safety

### PR DESCRIPTION
**Pull request by VarunSAthreya**
- Migrated the package to null safety standards. #6 
- Migrated from flutter package to dart package.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -4,14 +4,7 @@ version: 0.2.0
 homepage: https://github.com/sidsbrmnn/jwt_decode
 
 environment:
-  sdk: '>=2.10.0 <3.0.0'
+  sdk: ">=2.12.0 <3.0.0"
```
Comment (sidsbrmnn) [inline pubspec.yaml]: Fix the formatting. There's no code change here

**Review by sidsbrmnn — CHANGES_REQUESTED** [review]
Fix the formatting changes on pubspec.yaml file

**Comment by VarunAthreya** [comment]
Fixed the issue.

**Review by sidsbrmnn — APPROVED** [review]

**Comment by sidsbrmnn** [comment]
Thanks. Looks good

**Pull request merged**